roles/openshift_node_group/defaults/main.yml

```diff
@@ -41,7 +41,7 @@ l_openshift_prefix_dict:
   openshift-enterprise: 'ose-${component}'
 l_os_prefix: "{{ l_openshift_prefix_dict[openshift_deployment_type] }}"
```

Comment:
It appears l_os_prefix is not used any more?  Also l_openshift_prefix_dict ?

Same for other file and vars.

Reply:
Yea, removed them all.